videoConfProviders/PersistentChatVideoConfProvider.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Is this necessary?

Reply:
Not really, it came along as I copied another file; I've removed it now.